zproject/urls.py

```diff
@@ -822,7 +823,15 @@
 ]
 
 urls += [
-    path("self-hosted-billing/", self_hosting_auth_redirect, name="self_hosting_auth_redirect"),
+    path(
+        "self-hosted-billing/",
+        self_hosting_auth_redirect_endpoint,
+        name="self_hosting_auth_redirect_endpoint",
+    ),
+    rest_path(
+        "json/self-hosted-billing/",
```

Comment:
Is the trailing slash correct? I think our API URLs generally don't have that, and it's just the portico ones that do, but am not sure.

Reply:
Hmm I went for consistency with the original `/self-hosted-billing/` endpoint, but you're right, consistency with our usual API endpoint convention should probably override that. Tweaked to make this just `/json/self-hosted-billing` and pushed.